docs/gravitino-server-config.md

```diff
@@ -43,14 +43,19 @@ You can also specify filter parameters by setting configuration entries of the f
-| `gravitino.entity.store.maxTransactionSkewTimeMs` | The maximum skew time of transactions in milliseconds.                                                                                      | `2000`                           | No       | 0.3.0         |
-| `gravitino.entity.store.kv.deleteAfterTimeMs`     | The maximum time in milliseconds that deleted and old-version data is kept. Set to at least 10 minutes and no longer than 30 days. | `604800000`(7 days)              | No       | 0.3.0         |
+| Configuration item                                | Description                                                                                                                                                               | Default value                    | Required | Since version |
+|---------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------|----------|---------------|
+| `gravitino.entity.store`                          | Which storage implementation to use. Key-value pair storage and Relational storage is currently supported, the default value is `kv`, the optional value is `relational`. | `kv`                             | No       | 0.1.0         |
+| `gravitino.entity.store.kv`                       | Detailed implementation of KV storage. `RocksDB` storage is currently supported, and the implementation is `RocksDBKvBackend`.                                            | `RocksDBKvBackend`               | No       | 0.1.0         |
+| `gravitino.entity.store.relational`               | Detailed implementation of Relational storage. `MySQL` storage is currently supported, and the implementation is `MySQLBackend`.                                          | `MySQLBackend`                   | No       | 0.5.0         |
+| `gravitino.entity.store.relational.mysqlUrl`      | The database URL that the `MySQLBackend` needs to connect to.                                                                                                             | (none)                           | Yes      | 0.5.0         |
+| `gravitino.entity.store.relational.mysqlDriver`   | The jdbc driver name that the `MySQLBackend` needs to use. You should place the driver Jar package in the `${GRAVITINO_HOME}/libs/` directory.                            | (none)                           | Yes      | 0.5.0         |
+| `gravitino.entity.store.relational.mysqlUser`     | The username that the `MySQLBackend` needs to use when connecting the database.                                                                                           | (none)                           | No       | 0.5.0         |
```

Comment:
Is it a required config option?
`Yes if you use MySQLBackend as the entity store`

Reply:
I'm actually confused about whether `username` and `password` are necessary for the MySQL backend, because users can also connect  their MySQL backend without username and password if they set.

Reply:
JDBC catalog regards them as required. It's good to keep consistent with them.